azure-iot-device/setup.cfg

```diff
@@ -1,12 +1,29 @@
-#-------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-#--------------------------------------------------------------------------
+[bumpversion]
+current_version = 2.0.0-preview.7
```

Comment:
This is now maintained in two different spots as I understand it, which is concerning.

Reply:
ok let me clarify - in using pip bumpversion utility we declare this value in this file to be the version authority and it is henceforth updated by the bumpversion tool.  So changes to setup.py for a release will include a corresponding change in this file.  The tool uses regex for search and replace.  This flexible design accommodates incrementation of the arbitrarily labeled preview version as required.  But the trade-off is the tool must first know what to replace.  So here we are bootstrapping that value manually as a first time setup step.

Reply:
Also I updated the internal release doc with details about use of the bumpversion utility and linked to its reference page for more info.

Reply:
Does this mean that we can get rid of it after first time setup then?

---
In reply to: [301743275](https://github.com/Azure/azure-iot-sdk-python-preview/pull/134#discussion_r301743275) [](ancestors = 301743275)

Reply:
No it means the current_version key-value-pair will continue to live in setup.cfg and be updated by the bumpversion utility with each subsequent release.  So I'm trying to say this is the only time we have to deal with it - the tool will update the value thereafter as we direct it to.

If you have a specific reason not to include the [bumpversion] configuration setting in the .cfg file please lmk and I'll investigate if we can use the tool another way.  If you have any more questions about how bumpversion utility works please stop by my desk for a demo.